sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -145,11 +123,12 @@ private AnalyzeSentimentResult convertToAnalyzeSentimentResult(final DocumentSen
                         new RuntimeException(String.format(Locale.ROOT, "'%s' is not valid text sentiment.",
                             sentenceSentiment.getSentiment())));
                 }
-                SentimentConfidenceScorePerLabel confidenceScorePerSentence = sentenceSentiment.getSentenceScores();
+                com.azure.ai.textanalytics.implementation.models.SentimentConfidenceScorePerLabel
+                    confidenceScorePerSentence = sentenceSentiment.getSentenceScores();
 
                 return new SentenceSentiment(
                     sentimentLabel,
-                    new SentimentScorePerLabel(confidenceScorePerSentence.getNegative(),
+                    new SentimentConfidenceScorePerLabel(confidenceScorePerSentence.getNegative(),
```

Comment:
Is this the autogenerated one? why we use one here and in other places the one from `com.azure.ai.textanalytics.implementation.models.SentimentConfidenceScorePerLabel`

Reply:
It might be me and my limited knowledge of Java. It just seems strange how we are using for some things one and not other

Reply:
This one is not autogenerated one and you don't want to explore an autogenerated class to user. Another place that using autogenerated one is for internal mapping to non-autogenerated class.

Reply:
ugh so confusing. Thanks for clarifying it Shawn :)